js/layout-javascript/small-h-card-code.js

```diff
@@ -956,67 +949,10 @@ DynamicList.prototype.closeDetails = function() {
     }
 
     _this.$container.find('.dynamic-list-add-item').removeClass('hidden');
-  }, 300);
-};
 
-DynamicList.prototype.collapseElement = function(elementToCollapse) {
```

Comment:
The `expandElement` behaviour was removed in https://github.com/Fliplet/fliplet-widget-dynamic-lists/pull/673 which means the `collapseElement` behaviour is also no longer needed. Removing it in this PR ensures the focus is correctly moved back to the entry after an entry detail view is closed.